usaspending_api/recipient/v2/views/states.py

```diff
@@ -111,16 +110,29 @@ def get_all_states(year=None, award_type_codes=None, subawards=False):
             )
             .values("pop_state_code", "total", "distinct_awards", "outlay_total")
         )
-
         results = [
             {
                 "pop_state_code": row["pop_state_code"],
                 "total": row["total"],
                 "count": len(set(row["distinct_awards"].split(","))),
                 "total_outlays": row["outlay_total"],
             }
-            for row in list(queryset)
+            for row in list(fiscal_year_queryset)
         ]
+
+        # Get all other states and return `0` for their award count and totals
+        all_states_queryset = SummaryStateView.objects.all().distinct("pop_state_code").values("pop_state_code")
+        for row in all_states_queryset:
+            if row["pop_state_code"] not in [state["pop_state_code"] for state in results]:
```

Comment:
> [state["pop_state_code"] for state in results]

Can we break this out and define it outside of this `for` loop? This will recreate the list for each iteration of the loop